### PR TITLE
Request to upload the Hertie Coding Club website to the official Quarto Gallery

### PR DESCRIPTION
**Pull request by jurjoroa**
Request to upload the Hertie Coding Club website to the official Quarto Gallery

### CODE REVIEW

File: docs/gallery/gallery.yml [hunk 1]
```diff
@@ -206,10 +206,16 @@
       code: https://codeberg.org/jcolomb/fluidsyndicate
       thumbnail: websites/websites-fluidsyndicate.png  
     - title: Real World Data Science
-      subtitle: Ethical, impactful data science by the RSS
+      subtitle: Ethical, impactful DS by the RSS
```
Comment (cwickham) [inline docs/gallery/gallery.yml:209]: Would you mind reverting the change on this line? Seems unrelated.

File: docs/gallery/websites/websites-hertie-coding-club .png
Comment (cwickham) [inline docs/gallery/websites/websites-hertie-coding-club .png:1]: Looks like an erroneous space before `.png` is preventing this file being found. E.g. see the missing image in the preview at: https://deploy-preview-767--mystifying-jepsen-fa4396.netlify.app/docs/gallery/#websites

File: docs/gallery/gallery.yml (hunk 1 above)
Comment (jurjoroa) [reply docs/gallery/gallery.yml]: Done! I changed it (data science for DS) since it was outside the frame.

File: docs/gallery/websites/websites-hertie-coding-club .png
Comment (jurjoroa) [reply docs/gallery/websites/websites-hertie-coding-club .png:1]: Done! You were right! It had an extra space.

File: docs/gallery/gallery.yml (hunk 1 above)
Comment (cwickham) [reply docs/gallery/gallery.yml]: Ah I see the problem...I think that's something we need to fix on our end generally.

**Comment by cwickham** [comment]
Thanks @jurjoroa!
